src/store/types/index.ts

```diff
@@ -4,12 +4,12 @@ import {
   TRemoveFromArrayAction,
   TUpdateInArrayAction,
   TPushIntoArrayAction,
-  IForm,
```

Comment:
remember that TPayloadAction is an Union of all actions, so that we can have reducers taking in 
```ts
 action: TPayloadAction
```
as an argument, and the switch case will fail in case the object doesn't match
adding `IForm` as a possible type of PayloadAction would make things confusing